### PR TITLE
Add Lang team February update blog post

### PR DESCRIPTION
**Pull request by seanchen1991**

### CODE REVIEW

File: posts/inside-rust/2022-02-11-lang-team-feb-update.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
+---
+layout: post
+title: "Lang team February update"
+author: Sean Chen
+description: "Lang team February update"
+team: the lang team <https://lang-team.rust-lang.org/>
+---
+This week the lang team held its February planning meeting ([minutes]). We hold these meetings on the first Wednesday of every month.
```
Comment (badboy) [inline posts/inside-rust/2022-02-11-lang-team-feb-update.md:8]: Let's update the date of this blog post (file name) and then also change the wording here, as its not "this week" anymore.

**Review by badboy — CHANGES_REQUESTED** [review]

File: posts/inside-rust/2022-02-11-lang-team-feb-update.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+[active-initiatives]: https://lang-team.rust-lang.org/initiatives.html
+
+## Update from active initiatives
+
+This section details the updates submitted by the many active initiatives doing valuable work in the Rust project.
+Note that many of the more recent initiatives have dedicated repositories which contain other byproducts of the design process (e.g., evaluation documents, open design questions, etc).
+
+Unfortunately, no significant updates were reported by any of the active initiatives. 😞
+
+However, you can still check in on the progress of any active initiatives you're interested in by visiting the [active initiatives][active-initiatives] project board.
```
Comment (nikomatsakis) [inline posts/inside-rust/2022-02-11-lang-team-feb-update.md:28]: Let's rewrite this---

```suggestion
We didn't have any updates from initiatives this week and in fact are thinking of changing our "update" structure to integrate the updates into the regular triage meeting. For the time being, though, you can check in on the progress of any active initiatives you're interested in by visiting the [active initiatives][active-initiatives] project board, which is also a great way to get a summary of the major "goings on" of the lang team.
```

File: posts/inside-rust/2022-02-11-lang-team-feb-update.md (hunk 1 above)
Comment (nikomatsakis) [reply posts/inside-rust/2022-02-11-lang-team-feb-update.md:8]: I expect we would date the blog post from today.

**Review by nikomatsakis — CHANGES_REQUESTED** [review]

**Review by badboy — APPROVED** [review]

**Comment by badboy** [comment]
woops, I just wanted to change the date. let me fix that.

**Pull request merged**